src/main/flight/imu.c

```diff
@@ -525,12 +606,18 @@ static void imuCalculateEstimatedAttitude(float dT)
     }
 #endif
 
+    bool useGPSVelEF = sensors(SENSOR_GPS) && STATE(GPS_FIX) && gpsSol.flags.validVelNE && gpsSol.flags.validVelNE;
```

Comment:
gpsSol.flags.validVelNE && gpsSol.flags.validVelNE; 
Should be gpsSol.flags.validVelNE && gpsSol.flags.validVelD;

Reply:
Nice catch!